README.md

```diff
@@ -210,6 +210,41 @@ var HorizontalScroll = React.createClass({
 })
 ```
 
+## Tap / Click Elements inside IScroll
+
+To override the native scrolling iScroll has to inhibit some default browser behaviors, such as mouse clicks. See [iScroll manual](http://iscrolljs.com/) options.click and options.tap. The following is an example of including a clickable element.
+
+```js
+constructor(props) {
+    super(props);
+    this.refDict = [];
+}
+
+componentDidMount() {
+    this.refDict[0].addEventListener("tap", () => this.handleTap(this.refDict[0]));
+}
+
+handleTap(e) {
```

Comment:
Bind tap handler in constructor:

```javascript
  this.handleTap = this.handleTap.bind(this)
```

or use class props syntax:

```javascript
handleTap = (ev) => {
 ....
}
```

Then you dont need to call it `() => this.handleTap()`